Documentation/ceph-dashboard.md

```diff
@@ -111,3 +111,62 @@ rook-ceph-mgr-dashboard-external-https  NodePort    10.101.209.6     <none>
 In this example, port `31176` will be opened to expose port `8443` from the ceph-mgr pod. Find the ip address
 of the VM. If using minikube, you can run `minikube ip` to find the ip address.
 Now you can enter the URL in your browser such as `https://192.168.99.110:31176` and the dashboard will appear.
+
+If you have a cluster with an [nginx Ingress Controller](https://kubernetes.github.io/ingress-nginx/)
+and a Certificate Manager (e.g. [cert-manager](https://cert-manager.readthedocs.io/)) then you can create an
+Ingress like the one below. This example achieves four things:
+
+1. Exposes the dashboard on the Internet (using an reverse proxy)
+2. Issues an valid TLS Certificate for the specified domain name (using [ACME](https://en.wikipedia.org/wiki/Automated_Certificate_Management_Environment))
+3. Tells the reverse proxy that the dashboard itself uses HTTPS
+4. Tells the reverse proxy that the dashboard itself does not have a valid certificate (it is self-signed)
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: rook-ceph-mgr-dashboard
+  namespace: rook-ceph
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    kubernetes.io/tls-acme: "true"
+    nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
+    nginx.ingress.kubernetes.io/server-snippet: |
+      proxy_ssl_verify off;
+spec:
+  tls:
+   - hosts:
+     - rook-ceph.example.com
+     secretName: rook-ceph.example.com
+  rules:
+  - host: rook-ceph.example.com
+    http:
+      paths:
+      - path: /
+        backend:
+          serviceName: rook-ceph-mgr-dashboard
+          servicePort: https-dashboard
+```
+
+Customise the Ingress resource to match your cluster. Replace the example domain name 'rook-ceph.example.com'
```

Comment:
how about back ticks around `rook-ceph.example.com`?

Reply:
Added the backticks